src/main/kotlin/step2/Main.kt

```diff
@@ -0,0 +1,12 @@
+package step2
+
+import java.util.Scanner
+
+fun main() {
+    StringCalculator.parsing(
+        stringExpression = Scanner(System.`in`).nextLine(),
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
        stringExpression = readLine(),
```

Reply:
c7a92421113bfde2001a744525b95c0cb8363ef2
적용 완료하였습니다.